lib/src/proto/client.proto

```diff
@@ -0,0 +1,198 @@
+// Code generated by compiling proto template. DO NOT EDIT.
```

Comment:
this file is generated with gogoprotobuf specific tags, you can use a more simple one from https://github.com/centrifugal/centrifuge/blob/master/misc/proto/client.proto